kubernetes/linux/main.sh

```diff
@@ -1065,8 +1065,8 @@ if [ ! -f /etc/cron.d/ci-agent ]; then
       echo "*/5 * * * * root /usr/sbin/logrotate -s /var/lib/logrotate/ci-agent-status /etc/logrotate.d/ci-agent >/dev/null 2>&1" >/etc/cron.d/ci-agent
 fi
 
-setGlobalEnvVar AZMON_RESOURCE_OPTIMIZATION_ENABLED "${AZMON_RESOURCE_OPTIMIZATION_ENABLED}"
```

Comment:
lets make sure this doesnt cause any another regressions.

Reply:
https://github.com/microsoft/Docker-Provider/blob/662bef02d45fd503cbda9a848627096a8b0d1161/kubernetes/linux/main.sh#L1102  this value is defined here